### PR TITLE
PR review for https://github.com/wso2/docs-apim/pull/1793

### PR DESCRIPTION
**Pull request by Mariangela**
## Purpose
PR review for https://github.com/wso2/docs-apim/pull/1793

### CODE REVIEW

**Review by tharikaGitHub — APPROVED** [review]

File: en/docs/install-and-setup/setup/distributed-deployment/synchronizing-artifacts-in-a-gateway-cluster.md [hunk 1]
```diff
@@ -1,126 +1,129 @@
...
+    |gateway_labels|Specify the labels that the Gateway is going to subscribe to. Only the APIs with these labels will be pulled from the extension point and deployed.|
+    |artifact_retriever|  Specify the extension point. The default is `DBRetriever` where the artifacts are pulled from the database.|
+    |deployment_retry_duartion| Specify the retry duration in milliseconds to deploy artifacts if there is a failure in pulling them from the  extension through `deployment_retry_duartion`. The retry duration specified here will be exponentially increased by a progression factor of 2. Thereby, the duration will be progressed as 15s, 30s, 60s, 120s and so on if there are continuous failures. The retry duration is bounded by 1 hr.<html><div class="admonition note"><p class="admonition-title">Note</p><p> This`deployment_retry_duartion` is  applicable only in Asynchronous deployment (`data_retrieval_mode ="async"`) where the server will try to deploy the artifacts after it is started. </p></div></html>|
```
Comment (malinthaprasan) [inline en/docs/install-and-setup/setup/distributed-deployment/synchronizing-artifacts-in-a-gateway-cluster.md:92]: Is this a typo?

Comment (malinthaprasan) [inline en/docs/install-and-setup/setup/distributed-deployment/synchronizing-artifacts-in-a-gateway-cluster.md:92]: typo? deployment_retry_duartion

Comment (Mariangela) [reply en/docs/install-and-setup/setup/distributed-deployment/synchronizing-artifacts-in-a-gateway-cluster.md:92]: No. It is a config element. For example

```
[apim.sync_runtime_artifacts.gateway]
     gateway_labels =["Production and Sandbox","Label1","Label2"]
     artifact_retriever = "DBRetriever"
     deployment_retry_duartion = 15000
     data_retrieval_mode = "sync"
     save_artifacts_locally = false
```

Comment (Mariangela) [reply en/docs/install-and-setup/setup/distributed-deployment/synchronizing-artifacts-in-a-gateway-cluster.md:92]: Fixed in https://github.com/wso2/docs-apim/pull/1796

**Pull request merged**